README.md

```diff
@@ -23,83 +21,57 @@ Currently the operator installs the following products:
 - 3scale
 - Integreatly solution explorer
 
-## Prerequisites
+### managed-api
+
+- 3scale
+- RHSSO (both a cluster instance and a user instance)
+- Marin3r
 
-NOTE: Due to a change in how networking is configured for openshift in v4.4.6 (mentioned in the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator#supported-openshift-versions)) there is a limitation on the version of Openshift that RHMI can be installed on for BYOC clusters.
-Due to this change the use of integreatly-operator <= v2.4.0 on Openshift >= v4.4.6 is unsupported. Please use >= v2.5.0 of integreatly-operator for Openshift >= v4.4.6.
 
-- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v0.15.1.
+## Prerequisites
+
+- [operator-sdk](https://github.com/operator-framework/operator-sdk) version v1.2.0.
 - [go](https://golang.org/dl/) version 1.13.4+
 - [moq](https://github.com/matryer/moq)
-- [oc](https://docs.okd.io/3.11/cli_reference/get_started_cli.html#cli-reference-get-started-cli) version v3.11+
-- Access to an Openshift v4.2.0+ cluster
+- [oc](https://docs.okd.io/latest/cli_reference/openshift_cli/getting-started-cli.html) version v4.6+
+- Access to an Openshift v4.6.0+ cluster
 - A user with administrative privileges in the OpenShift cluster
-- AWS account with permissions to create S3 buckets
 
-After installation, the following commands must be run to avoid a known [issue](https://github.com/matryer/moq/issues/98) related to the Moq package:
```

Comment:
removing this section as I don't think this is an issue anymore. open to correction